runtime.json

```diff
@@ -51,23 +51,27 @@
                 "key": "port",
                 "value": "55555"
             },
+            {
```

Comment:
This should be removed if there is no some special reason to have it here, because it is not fully implemented in gen scripts.